operators/olm_v1/arch/olmv1-operator-controller.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="olmv1-operator-controller"]
+= Operator Controller (Technology Preview)
+include::_attributes/common-attributes.adoc[]
+:context: olmv1-operator-controller
+
+toc::[]
+
+Operator Controller is the central component of {olmv1-first} and consumes the other three {olmv1} components: RukPak, Deppy, and catalogd. It extends Kubernetes with an API through which users can install Operators and extensions.
```

Comment:
```suggestion
Operator Controller is the central component of {olmv1-first} and consumes the other three {olmv1} components: RukPak, Deppy, and catalogd. It extends Kubernetes with the `Operator` API through which users can install Operators and extensions.
```

Reply:
Sugg: Since there is a specific name for this, it could be helpful to put it here instead of just "an API".

Reply:
There's a little bit of a dance going on here because there is an older version of `Operator` API as well (see NOTE [here](https://65963--docspreview.netlify.app/openshift-enterprise/latest/operators/olm_v1/arch/olmv1-operator-controller.html#olmv1-operator-api))